### ISSUE TITLE
Permission denied when using kind example circle ci job

### ISSUE THREAD

**Issue by omerlh** [helm/chart-testing#79]
**Is this a request for help?**: ues

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): bug report

**What happened**:
I copied the setting from the kind example to my chart repository (hope I can make it public tomorrow), modify the required settings and push the changes. When running on circle ci, the job `install-charts` failed with the following command:
```
chmod: changing permissions of ‘kind’: Operation not permitted
```
I think this is a permission issue - kind is downloaded on golang docker image, meaning it is downloaded as root, which causes `chmod` to fail. 
Installing kind using curl seem to solve the problem:
```
curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/0.0.1/kind-linux-amd64
```
**What you expected to happen**:
Test to run successfully (or with issues related to my code)

**How to reproduce it** (as minimally and precisely as possible):
Run the kind example on circle ci

**Anything else we need to know**:
This is an amazing project, thank you so much!

**Comment by unguiculus** [comment]
cc @rimusz

**Comment by rimusz** [comment]
@omerlh did you try to remove `chmod +x kind`? as `kind` binary is already executable.
`docker run --rm -it -v "$(pwd)":/go/bin golang go get sigs.k8s.io/kind && sudo mv kind /usr/local/bin/`

**Comment by omerlh** [comment]
I'll try to do maybe later, but this might introduce another challenge as I can't execute kind without sudo. Unless this is intended?

**Comment by rimusz** [comment]
Fix is in #80
